examples/common.yaml

```diff
@@ -132,7 +132,7 @@ openstack::swift::hash_suffix: 'pop-bang'
 
 ######## Nova
 
-openstack::nova::libvirt_type: 'kvm'
+openstack::nova::libvirt_type: 'qemu'
```

Comment:
Greetings,

I will take this up.

One question although, should this work go into puppetlabs/openstack or
stackforge/openstack.
Being new I am still a bit confused on that.

regards
/sanjay

On Wed, Jun 10, 2015 at 3:17 AM, Colleen Murphy <notifications@github.com>
wrote:

> In examples/common.yaml
> <https://github.com/puppetlabs/puppetlabs-openstack/pull/199#discussion_r32068357>
> :
>
> > @@ -132,7 +132,7 @@ openstack::swift::hash_suffix: 'pop-bang'
> >
> >  ######## Nova
> >
> > -openstack::nova::libvirt_type: 'kvm'
> > +openstack::nova::libvirt_type: 'qemu'
>
> This change needs to be made in examples/allinone.yaml as well
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/puppetlabs/puppetlabs-openstack/pull/199/files#r32068357>
> .
>



-- 
Sanjay Upadhyay
http://saneax.blogspot.com